src/scss/attendee/layer.scss

```diff
@@ -34,4 +34,6 @@
 // Major application or page wide z-index values should follow here
 // Minor ones constrained to their own component can be provided in separate files
 $-z-index--baseline: 1000;
+$-z-index--header: 50;
```

Comment:
Please make this relative to the baseline. It should be an unbroken tree that only has the root with a hard-coded value.

Reply:
hey, so would something like `level($-z-index--baseline + 100)` work? or am I not getting you right?
[1ec7d1e](https://github.com/sparkletown/sparkle/pull/2973/commits/1ec7d1e6854fb0c34f6633c95e8d4ec95fe9603c)